Makefile

```diff
@@ -31,12 +32,17 @@ check-test-project:
 
 all-dockerized: run-node build-builder
 	docker run --rm --network host \
-		--mount type=bind,source="$(shell pwd)",target=/code \
+		--user "$(shell id -u):$(shell id -g)" \
+		--volume "$(shell pwd)":/code \
```

Comment:
nit: Wouldn't `$(pwd):/code` work as well?

Reply:
It wouldn't, something with it happening in a Makefile. `$pwd` is empty and you need to do this.